CHANGELOG.md

```
@@ -4,11 +4,19 @@ All notable changes to DotAIOS will be documented in this file.
 
 ## [Unreleased]
 
-### Changed
-- The post-publication source is now versioned as 2.0.14 (unreleased) so it
-  cannot be mistaken for the immutable 2.0.13 package published from
-  `77350441b706c816c5c5e5e63c175962632e415f`. Customer setup docs remain pinned
-  to the available 2.0.13 package until 2.0.14 is published.
+## [2.0.14] - 2026-08-30
+
+### Added
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)